contrib/babelfishpg_tsql/src/guc.c

```diff
@@ -1517,16 +1516,6 @@ define_escape_hatch_variables(void)
 							 PGC_USERSET,
 							 GUC_NOT_IN_SAMPLE | GUC_DISALLOW_IN_FILE | GUC_DISALLOW_IN_AUTO_FILE,
 							 NULL, NULL, NULL);
-
-	DefineCustomEnumVariable("babelfishpg_tsql.escape_hatch_unique_constraint",
```

Comment:
I have one question , we explicit the usage of escape_hatch_unique_constraint in doc : https://babelfishpg.org/docs/usage/escape_hatches/ , and if we are deleting this guc, we'll have to make sure it's continuous and safe for customer.

Reply:
Yes, changed the behaviour to send out a deprecate message instead of error [link](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/0b61d9ed5d388d482728ef1f7de33b48c2d22159/contrib/babelfishpg_tsql/sql/sys_procedures.sql#L141)
sp_babelfish_configure which is the suggested way to set escape hatches will not throw an error now. But set_config() will still error out.